readme.md

```diff
@@ -0,0 +1,47 @@
+[![Run PHPUnit and PHPCS](https://github.com/Automattic/maintenance-mode-wp/actions/workflows/integrate.yml/badge.svg)](https://github.com/Automattic/maintenance-mode-wp/actions/workflows/integrate.yml)
+
+# Maintenance Mode
+
+Shut down your site for a little while and do some maintenance on it!
+
+Usage:
+
+ - Add a template to your theme's root folder called `template-maintenance-mode.php`.
+ - This should be a simple HTML page that should include the message you want to show your visitors.
+ - Note: the template should include `wp_head()` and `wp_footer()` calls.
+ - Add the VIP_MAINTENANCE_MODE constant to your theme and set to `true`.
+
+## Installation
+
+1. Install the plugin.
+1. Set the `VIP_MAINTENANCE_MODE` constant to true.
+1. Activate.
+
+## Default template screenshots
+
+twentytwentyone:  
+![twentytwentyone](screenshot-twentytwentyone.png)
```

Comment:
Accessibility-wise, should we change the Alt description to something like

`![Screenshot of Maintenance Mode active on the Twenty Twenty One theme](screenshot-twentytwentyone.png)`